lib/fetch/index.js

```diff
@@ -92,13 +92,13 @@ export function fetchTalk(args) {
 }
 
 // TODO: Test? lol
```

Comment:
No, because the ticket doesn't include any information on what bits need attention.

I think this is a great comment. It really highlights what the problem is with this code.